## 01_serverlist.dpatch by evo, edited by bartm
##
## DP: Add debian and chillfactory irc servers.
## DP: Make Debian servers default.
--- a/src/common/servlist.c
+++ b/src/common/servlist.c
@@ -46,6 +46,12 @@
 
 static const struct defaultserver def[] =
 {
+	{"Debian Servers", 0, "#debian"},
+	{0,                     "irc.debian.org"},
+
+	{"Ubuntu Servers", 0, "#ubuntu"},
+	{0,                     "irc.ubuntu.com"},
+
 	{"2600net",	0},
 	{0,			"irc.2600.net"},
 
@@ -924,7 +930,7 @@
 				free (net->encoding);
 				net->encoding = strdup (def[i].charset);
 			}
-			if (g_str_hash (def[i].network) == 0x8e1b96f7)
+			if (g_str_hash (def[i].network) == 0x37473b25)
 				prefs.slist_select = j;
 			j++;
 		} else
